config/gazebo9_bionic.yaml

```diff
@@ -0,0 +1,6 @@
+name: gazebo9bionic
+method: http://ppa.launchpad.net/j-rivero/gazebo9bionic/ubuntu
+suites: [bionic]
+component: main
+architectures: [amd64, arm64, armhf, i386, source]
+filter_formula: Package (% gazebo9), $Version (% 9.0.0+dfsg5-3ubuntu1+ppa2)
```

Comment:
Do we need the libgazebo9* packages as well? Would you accept a change that uses exact package names for clarity, matching the format used in #94?

Reply:
2e1b8d0